proto/nibiru/perp/v2/genesis.proto

```diff
@@ -35,7 +35,27 @@ message GenesisState {
     ];
   }
 
-  repeated GenesisMarketLastVersion market_last_versions = 8
+  message Discount {
+    string fee = 1 [
+      (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+      (gogoproto.nullable) = false
+    ];
+    string volume = 2 [
+      (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+      (gogoproto.nullable) = false
+    ];
+  }
+
+  repeated Discount global_discount = 8 [ (gogoproto.nullable) = false ];
```

Comment:
if repeated, is global_discounts?

Reply:
yes, global discount is a discount that applies to everybody and it's like a tuple of volume and fee associated with that volume.